source/_docs/dns.md

```diff
@@ -1,12 +1,161 @@
+
+  If your site doesn't have these advanced needs, there are free and inexpensive options outside of Pantheon. We recommend considering your domain registrar's DNS services, [Amazon Route 53](https://aws.amazon.com/route53/){.external}, [Google Cloud DNS](https://cloud.google.com/dns/){.external}, or [Cloudflare](https://woorkup.com/cloudflare-dns/){.external}.
+  </div>
+</div>
+
+### Why does the `www` subdomain redirect to the bare domain?
+
+Some DNS providers provide a default `CNAME` record for `www` pointing to `@` (the bare domain). Remove these records and replace them with the records suggested by the Pantheon Site Dashboard.
+
+### What are `AAAA` records, and do I need them?
+
+**AAAA** or "quad-A" records are used to assign [IPv6](https://en.wikipedia.org/wiki/IPv6){.external} addresses to domain names. While most of the internet still uses IPv4, the address pool has been exhausted and IPv6 became the standard as of July 14th, 2017.
+
+Pantheon provides IPv6 addresses for you to assign `AAAA` records to. If your DNS service does not support `AAAA` you can simply omit this records for now, but consider asking your provider to add functionality, as the need for IPv6 will only increase in the future.
+
+Some DNS service providers require expanded AAAA records. You can retrieve the expanded version for each AAAA record by using the [IPv6 conversion tool](http://ipv6-literal.com/){.external} and copying the "IPv6 longest" result.
+
+
+### What about my `MX` records for email?
+
+Pantheon does not provide email services. Make sure your DNS records include an `MX` reacord that points to a subdomain (like `mail`), which in turn has an `A` or `CNAME` record pointing it to your email provider.
+
+### What is the difference between an `A` and `CNAME` record?
+
+An `A` record points a domain name to an IPv4 address. A `CNAME` record points a domain name to *another* domain, but *does not* redirect to it.
+
+Pantheon will provide both `A` and `CNAME` values, depending on the domains provided:
+
+![DNS Values provided by the Pantheon Site Dashboard](/source/docs/assets/images/dashboard/dns-values.png)
+
+If your DNS provider doesn't allow `CNAME` records, you can use `nslookup` to determine the IP address for the domain value, and then supply that as an `A` record:
+
+```bash
+$ nslookup live-anita-drupal.pantheonsite.io
+Server:         192.168.1.1
+Address:        192.168.1.1#53
+
+Non-authoritative answer:
+live-anita-drupal.pantheonsite.io       canonical name = fe1.edge.pantheon.io.
+Name:   fe1.edge.pantheon.io
+Address: 23.185.0.1
+Name:   fe1.edge.pantheon.io
+Address: 2620:12a:8001::1
+Name:   fe1.edge.pantheon.io
+Address: 2620:12a:8000::1
+```
+
+In the example above, we can create an `A` record for `www` set to `23.185.0.1` to substitute the `CNAME` record.
```

Comment:
Consider adding a see also section that links to launch guide and platform domains doc and maybe redirect doc?

Reply:
Done, see below.